build.gradle

```
@@ -15,97 +15,130 @@
  * along with Baritone.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-group 'baritone'
-version '1.2.17'
+allprojects {
+    apply plugin: 'java'
+    apply plugin: "xyz.wagyourtail.unimined"
+    apply plugin: "maven-publish"
+
+    archivesBaseName = rootProject.archives_base_name
+    def vers = 'git describe --always --tags --dirty'.execute().text.trim()
+    if (!vers.startsWith("v")) {
+        println "git doesn't appear to be installed!"
```

Actually, why not try to suggest the thing I mean again? (And this time I applied it locally before commenting)
```suggestion
    def vers;
    try {
        vers = 'git describe --always --tags --match v* --dirty'.execute().text.trim()
    } catch (IOException ignored) {
        vers = ""
        println "git doesn't appear to be installed!"
    }
    if (vers.equals("")) {
        println "couldn't find version tag!"
```

For some reason trying to quote the glob pattern makes it not find anything. Seems like the quotes become part of the pattern.